src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
```

Comment:
Now the name is clear in the context of this file, but if you use this interface in many places, confusion may arise, so it is better to give more precise names, for example `TodoInterface`/`ITodo` or `TodoType`/`TTodo` (if it will be a type).